Tests/RxSwift/MapAtTests.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  MapToKeyPath.swift
+//  RxSwiftExt-iOS
+//
+//  Created by Michael Avila on 2/8/18.
+//  Copyright © 2018 RxSwiftCommunity. All rights reserved.
+//
+
+import XCTest
+
+import RxSwift
+import RxSwiftExt
+import RxTest
+
+struct Person {
+    let name: String
+}
+
+class MapAtTests: XCTestCase {
+
+    let people: [Person] = [
+        Person(name: "Bart"),
+        Person(name: "Lisa"),
+        Person(name: "Maggie")
+    ]
+
+    private var observer: TestableObserver<String>!
+
+    override func setUp() {
+        let scheduler = TestScheduler(initialClock: 0)
+        observer = scheduler.createObserver(String.self)
+
+        _ = Observable.from(people)
+            .mapAt(\.name)
+            .subscribe(observer)
+
+        scheduler.start()
+    }
+
+    func testReplaceWithResultCount() {
```

Comment:
What do you think about the naming of the tests? I'm just thinking out loud here if the name is proper to what its testing

testMapAtKeypathCount()
testMapAtKeypathValues() 

Anyways - I don't really mind leaving as is, it doesn't affect anything much, would love your opinion :)